src/js/models/maps/Map.js

```diff
@@ -395,7 +395,8 @@ define([
       getLayerGroups() {
         if (this.has("layerCategories")) {
           return this.get("layerCategories").getMapAssets();
-        } else if (this.has("layers")) {
+        }
+        if (this.has("layers")) {
           return [this.get("layers")];
```

Comment:
@alonakos curious why you changed this from `else if` to `if`? Was this part of the bug fix?